plugin/pkg/admission/storageclass/default/admission.go

```diff
@@ -88,7 +88,7 @@ func (a *claimDefaulterPlugin) SetInternalClientSet(client internalclientset.Int
 	}
 }
 
-// Validate ensures an authorizer is set.
+// Validate ensures clientset and store are set.
```

Comment:
Did not try godoc, but here's some good suggestion for `golint` for pkg/scheduler; we can include golint into verify tool chains.

```
golint plugin/pkg/scheduler/
plugin/pkg/scheduler/scheduler.go:43:6: exported type PodConditionUpdater should have comment or be unexported
plugin/pkg/scheduler/scheduler.go:53:1: exported method Scheduler.StopEverything should have comment or be unexported
plugin/pkg/scheduler/scheduler.go:57:1: comment on exported type Configurator should be of the form "Configurator ..." (with optional leading article)
plugin/pkg/scheduler/scheduler.go:83:1: comment on exported type Config should be of the form "Config ..." (with optional leading article)
plugin/pkg/scheduler/scheduler.go:143:1: receiver name s should be consistent with previous receiver name sched for Scheduler
plugin/pkg/scheduler/scheduler.go:147:1: receiver name s should be consistent with previous receiver name sched for Scheduler
```

 The best case is to combine it into a more meaningful PR, but it's hard for most of case. I think the bottleneck is the reviewer pool for such kind of doc improvement :). So, it requires the original reviewer to cover such kind of issue; for the reviewer pool, suggest to add more reviewers :).